gradle/libs.versions.toml

```diff
@@ -33,6 +33,8 @@ androidx-activity-compose = "1.10.0"
 androidx-activity-core = "1.10.0"
 androidx-annotation = "1.9.1"
 androidx-composeBom = "2025.01.01"
+androidx-ui-alpha = "1.8.0-alpha08"
+#todo: remove this alpha when stable compose version bumps to 1.8
```

Comment:
Not a big deal, but sometimes I get confused if the comment refers to the thing that is above or below it. When we add a comment/doc on a function or a class we always do it above the thing we are commenting on since it's impractical to do it at the end of the class instead. If we can keep this consistent in all places it helps with not getting confused sometimes in those comments that are on a single line instead of on a bigger class/function.
The suggestion is to change it to this:
```
#todo: remove this alpha when stable compose version bumps to 1.8
androidx-ui-alpha = "1.8.0-alpha08"
```